### PR TITLE
Remove common protos

### PR DESCRIPTION
**Pull request by michaelbausor**
Removes common protos and adds dependency on google/common-protos

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,14 +22,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Google\\Api\\": "src/Api",
             "Google\\ApiCore\\": "src/ApiCore",
```
Comment (dwsupplee) [inline composer.json]: If you're open to it, we should be able to put everything in the `ApiCore` directory right into `src` now and replace this line with `"Google\\ApiCore\\": "src"`

Comment (michaelbausor) [reply composer.json]: Was just thinking the same thing actually :-)
Done. PTAL at src/Version, which needed a change.

Comment (dwsupplee) [reply composer.json]: Great catch 👍

Comment (michaelbausor) [reply composer.json]: Should I do the same in tests?

File: composer.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
         "grpc/grpc": "^1.4",
         "google/protobuf": "^3.5.1",
```
Comment (dwsupplee) [inline composer.json]: I noticed [here](https://github.com/googleapis/google-cloud-php/pull/1517/files#diff-ad6978f3626f02acfb16dbe103da2356R9) we are requiring a higher version of protobuf, would there be any harm in doing the same here?

File: composer.json (hunk 1 above)
Comment (dwsupplee) [reply composer.json]: Yes, SGTM :)

File: composer.json (hunk 2 above)
Comment (michaelbausor) [reply composer.json]: SGTM, done

**Pull request merged**